docs/developer-guide/plugins.md

```diff
@@ -14,36 +14,47 @@ We recommend your custom rules adhere to our [rule conventions](rules.md) for:
 
 ## The anatomy of a plugin
 
+Let's say we write a plugin forbidding certain word (e.g., "foo") within a selector.
+The plugin's implementation would be:
```

Comment:
This example plugin disallows the word "foo" in selectors:

Reply:
Thanks. Fixed via 3288290af.